package.json

```diff
@@ -22,13 +22,14 @@
     "node",
     "setup"
   ],
-  "author": "",
+  "author": "Netherlands eScience Center",
   "license": "Apache-2.0",
   "dependencies": {
     "@actions/core": "^1.2.6",
     "js-yaml": "^4.1.0"
   },
   "devDependencies": {
+    "@actions/exec": "^1.0.4",
```

Comment:
Please move to dependencies as exec is used on runtime, not during development (`npm run *`).

Reply:
yep